mlflow/gateway/runner.py

```diff
@@ -95,6 +95,8 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 def run_app(config_path: str, host: str, port: int, workers: int) -> None:
     config_path = os.path.abspath(os.path.normpath(os.path.expanduser(config_path)))
+    if os.path.islink(config_path):
+        config_path = os.readlink(config_path)
```

Comment:
Do we want to support the case that the link file points to another link file ?

Reply:
I think it would be smart: I'll just recursively keep checking until we get to the actual file

Reply:
but I think it might be complex , in this case we need to monitor all files inside the link list.

e.g.: A -> B -> C -> D,   we need to monitor changes on file A,B,C,D. 

I think this PR we can only support current case, and in follow-up PR we can support complex case if it needed.

Reply:
Also note that, current PR code:

```
    if os.path.islink(config_path):
        config_path = os.readlink(config_path)
```

If we change the link file to make it link to a new config file, your PR code cannot capture this case file change.